make/test.mk

```diff
@@ -44,14 +44,14 @@ upload-codecov-report:
 # End-to-end Tests
 #
 ###########################################################
-
 .PHONY: test-e2e
 test-e2e:  e2e-setup
 	sed -e 's|REPLACE_IMAGE|${IMAGE_NAME}|g' ./deploy/operator.yaml  | oc apply -f -
 	# This is hack to fix https://github.com/operator-framework/operator-sdk/issues/1657
 	echo "info: Running go mod vendor"
 	go mod vendor
-	operator-sdk test local ./test/e2e --no-setup --namespace $(TEST_NAMESPACE) --go-test-flags "-v -timeout=15m"
+	oc login -u system:admin
+	operator-sdk test local ./test/e2e --namespace $(TEST_NAMESPACE) --up-local --go-test-flags "-v -timeout=15m"
```

Comment:
@tinakurian what's the need of these changes ?? Also, removal of `--no-setup` is intentional ??

Reply:
Yeah... `oc login -u system:admin` looks suspicious. Maybe our SA is missing some permissions? cc: @xcoulon

Reply:
Sorry, this was from an earlier commit. It's now reverted back to what we had!